src/contracts/core/AllocationManager.sol

```diff
@@ -222,25 +214,25 @@ contract AllocationManager is
                     PendingFreeMagnitude storage pendingFreeMagnitude =
                         _pendingFreeMagnitude[operator][strategies[i]][index];
 
-                    // Reached pendingFreeMagnitude/deallocation that is completable and not within slashability window,
+                    // Reached deallocation that is completable and, therefore, not slashable,
                     // therefore older deallocations will also be completable. Since this is ordered by completableTimestamps break loop now
-                    if (pendingFreeMagnitude.completableTimestamp >= uint32(block.timestamp)) {
+                    if (block.timestamp >= pendingFreeMagnitude.completableTimestamp) {
```

Comment:
oh this is an interesting use - I should test.

Reply:
whoops i just rewrote this and didn't realize i fixed a bug